.travis.yml

```diff
@@ -68,7 +68,8 @@ install:
   - mkdir -p "$HOME/downloads"
   - mkdir -p "$TRAVIS_BUILD_DIR/deps/cache/"
   - find "$HOME/downloads" -type f -exec cp -v {} "$TRAVIS_BUILD_DIR/deps/cache/" \;
-  - on_host virtualenv --python=python3 jmvenv
+  - on_host do_on linux ./install.sh --develop --python=python3 --with-qt
```

Comment:
i don't believe joinmarket-qt has any tests at the moment so maybe it would save some time and bandwidth to omit the `--with-qt` option?

Reply:
Correct there are no automated tests of JM-Qt

Reply:
I disagree.  We have one automated test for JM-Qt which is "JM-Qt is successfully installed by passing `--with-qt` to `install.sh` :)

Reply:
Oh I realize you mean actual bandwidth and as a result decrease test times.
Do we not cache the dependencies for the gui?
If not then it's a bug in our travis setup, and I agree we should remove `--with-qt` from here if so.

Reply:
So yes, we do cache pip stuff incl. gui e.g. 
https://travis-ci.org/fivepiece/joinmarket-clientserver/jobs/606396223#L1537-L1539

Also, the travis test for ubuntu is done in about 7 minutes while the osx one is much longer (now that tests on osx are enabled), so further the gui install isn't slowing us down.

But -- if you still think we should stop installing Qt in travis, I'll make the required changes.

Reply:
Any updates on this one @takinbo ?

Reply:
LGTM